--- /tmp/64bit_regression_tests	2020-07-24 18:34:51.811830486 +0000
+++ 64bit_regression_tests	2020-07-24 18:36:44.827779374 +0000
@@ -87,7 +87,7 @@
  <type>union myunion</type>
 -<full_lhs>byte_extract_little_endian\(u, 0ll?, .*int.*\)</full_lhs>
 +<full_lhs>byte_extract_little_endian\(u, 0l?l?, .*int.*\)</full_lhs>
- <full_lhs_value>\d+ll?</full_lhs_value>
+ <full_lhs_value binary="[01]+">\d+ll?</full_lhs_value>
  <value>\{ \.i=\d+ll? \}</value>
  <value_expression>
 --- cbmc-5.12.orig/regression/cbmc/byte_update11/main.c
@@ -147,14 +147,14 @@
  --harness-type call-function --function test --treat-pointer-as-array arr --associated-array-size arr:sz
  ^EXIT=0$
  ^SIGNAL=0$
--\[test.pointer_dereference.1\] line \d+ dereference failure: pointer NULL in arr\[\(signed( long)* int\)i\]: SUCCESS
--\[test.pointer_dereference.2\] line \d+ dereference failure: pointer invalid in arr\[\(signed( long)* int\)i\]: SUCCESS
+-\[test.pointer_dereference.1\] line \d+ dereference failure: pointer invalid in arr\[\(signed( long)* int\)i\]: SUCCESS
+-\[test.pointer_dereference.2\] line \d+ dereference failure: pointer NULL in arr\[\(signed( long)* int\)i\]: SUCCESS
 -\[test.pointer_dereference.3\] line \d+ dereference failure: deallocated dynamic object in arr\[\(signed( long)* int\)i\]: SUCCESS
 -\[test.pointer_dereference.4\] line \d+ dereference failure: dead object in arr\[\(signed( long)* int\)i\]: SUCCESS
 -\[test.pointer_dereference.5\] line \d+ dereference failure: pointer outside dynamic object bounds in arr\[\(signed( long)* int\)i\]: SUCCESS
 -\[test.pointer_dereference.6\] line \d+ dereference failure: pointer outside object bounds in arr\[\(signed( long)* int\)i\]: SUCCESS
-+\[test.pointer_dereference.1\] line \d+ dereference failure: pointer NULL in arr\[(\(signed( long)* int\))?i\]: SUCCESS
-+\[test.pointer_dereference.2\] line \d+ dereference failure: pointer invalid in arr\[(\(signed( long)* int\))?i\]: SUCCESS
++\[test.pointer_dereference.1\] line \d+ dereference failure: pointer invalid in arr\[(\(signed( long)* int\))?i\]: SUCCESS
++\[test.pointer_dereference.2\] line \d+ dereference failure: pointer NULL in arr\[(\(signed( long)* int\))?i\]: SUCCESS
 +\[test.pointer_dereference.3\] line \d+ dereference failure: deallocated dynamic object in arr\[(\(signed( long)* int\))?i\]: SUCCESS
 +\[test.pointer_dereference.4\] line \d+ dereference failure: dead object in arr\[(\(signed( long)* int\))?i\]: SUCCESS
 +\[test.pointer_dereference.5\] line \d+ dereference failure: pointer outside dynamic object bounds in arr\[(\(signed( long)* int\))?i\]: SUCCESS
